googleapis/google-cloud-node
# migrate code from googleapis/nodejs-storage-transfer

- feat: initial stub of library
- feat!: initial generation of nodejs-storage-transfer library (#2)
- chore: release 1.0.0 (#3)
- fix(deps): google-gax v2.24.1 (#7)
- chore: release 1.0.1 (#8)
- feat: turns on self-signed JWT feature flag (#9)
- chore: release 1.1.0 (#10)
- chore(deps): update dependency mocha to v9 (#4)
- chore(deps): update dependency sinon to v11 (#5)
- chore: disable renovate dependency dashboard (#1194) (#12)
- fix(build): set default branch to main (#14)
- chore: release 1.1.1 (#15)
- chore: relocate owl bot post processor (#17)
- build(node): update deps used during postprocessing (#1243) (#22)
- chore(deps): update dependency @types/node to v16 (#23)
- chore(deps): update dependency sinon to v12 (#25)
- docs(samples): add example tags to generated samples (#26)
- Update the quickstart sample to be in line with other languages (#24)
- build: add generated samples to .eslintignore (#27)
- docs(node): support "stable"/"preview" release level (#1312) (#30)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#31)
- test(nodejs): remove 15 add 16 (#1322) (#33)
- chore: add api_shortname and library_type to repo metadata (#29)
- chore: update v2.12.0 gapic-generator-typescript (#37)
- chore(deps): update dependency sinon to v13 (#39)
- docs(nodejs): version support policy edits (#1346) (#41)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#43)
- feat: Increase STS Sample Coverage (#42)
- chore: update v2.14.2 gapic-generator-typescript (#48)
- feat: add support for Agent Pools (#51)
- chore(main): release 1.2.0 (#45)
- build(node): update client library version in samples metadata (#1356) (#57)
- chore(deps): update dependency sinon to v14 (#62)
- build!: update library to use Node 12 (#64)
- chore(deps): update dependency @google-cloud/storage to v6 (#66)
- chore(main): release 2.0.0 (#65)
- chore(deps): update dependency jsdoc-region-tag to v2 (#70)
- chore(deps): update dependency jsdoc-fresh to v2 (#69)
- feat(samples): Add POSIX & Manifest samples (#67)
- feat: support regapic LRO (#75)
- chore(main): release 2.1.0 (#76)
- chore(deps): update dependency linkinator to v4 (#80)
- fix: better support for fallback mode (#83)
- fix: change import long to require (#85)
- chore(main): release 2.1.1 (#86)
- fix: do not import the whole google-gax from proto JS (#1553) (#89)
- fix: allow passing gax instance to client constructor (#90)
- chore(main): release 2.1.2 (#91)
- chore(deps): update dependency uuid to v9 (#92)
- fix: preserve default values in x-goog-request-params header (#94)
- test: use fully qualified request type name in tests (#111)
- chore(main): release 2.2.0 (#95)
- samples: Add AWS S3-compatible Sample (#118)
- fix(deps): use google-gax v3.5.2 (#121)
- chore(deps): update dependency @types/node to v18 (#119)
- chore(deps): update dependency jsdoc to v4 (#124)
- fix: regenerated protos JS and TS definitions (#125)
- chore(main): release 2.2.1 (#123)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
